test/auth_SUITE.erl

```diff
@@ -89,7 +100,18 @@ mqtt_config(ssl_user) ->
                      {allow_anonymous, false}]};
 mqtt_config(no_ssl_user) ->
     {rabbitmq_mqtt, [{ssl_cert_login,  false},
-                     {allow_anonymous, false}]}.
+                     {allow_anonymous, false}]};
+mqtt_config(client_id_propagation) ->
+    {rabbitmq_mqtt, [{ssl_cert_login,  true},
+                     {allow_anonymous, true}]}.
+
+auth_config(client_id_propagation) ->
+    {rabbit, [
+            {auth_backends, [rabbit_auth_backend_mqtt_mock]}
```

Comment:
I don't see this module anywhere. The test [fails with an `undef`](https://gist.github.com/michaelklishin/41c3416ba0bcded2fae42112c9ba9400).

Reply:
Added and pushed, sorry about that.